cmd/sti/main.go

```diff
@@ -191,6 +192,56 @@ func newCmdBuild(cfg *api.Config) *cobra.Command {
 	return buildCmd
 }
 
+func newCmdRebuild(cfg *api.Config) *cobra.Command {
+	buildCmd := &cobra.Command{
+		Use:   "rebuild <image>",
```

Comment:
@bparees i think the "rebuild" means that you want to output the same image (it will re-tag), but I also think that having optional new image name will not hurt. Will add that